sdk/ai/Azure.AI.Agents/src/Custom/AIClientModelFactory.cs

```diff
@@ -18,7 +18,7 @@ namespace Azure.AI.Projects;
 
 public static partial class AIClientModelFactory
```

Comment:
I know this is not directly related to your split, but the name of this public class seems very generic... Should it not be called AIAgentsClientModelFactory?